packages/kbn-doc-links/README.md

```diff
@@ -0,0 +1,3 @@
+# `@kbn/doc-links` — Kibana documentation links registry
```

Comment:
Please, remember to assign this package to the appropriate CODEOWNERS

Reply:
We're willingly not putting `core` as a team owner of the module, to avoid reviews every time any team adds a link.

Reply:
@elastic/kibana-docs do you think your team should be set as owners of this new module?

Reply:
Yes, that seems reasonable, thanks!